### ISSUE TITLE
Programmatic API: configFile option is not honored

### ISSUE THREAD

**Issue by kenrick95** [lwsjs/local-web-server#115]
When using the programmatic API of this package, I followed the example and documentation from [the wiki](https://github.com/lwsjs/local-web-server/wiki/API-reference).

On that page, it listed the option `configFile` as a parameter to `LocalWebServer.listen()`

So, I expect this piece of codes to work as intended (options are read from `configFile`):

```js
const LocalWebServer = require('local-web-server')
const localWebServer = new LocalWebServer()
const server = localWebServer.listen({
  configFile: './path-to-file/lws.config.js'
})

server.close() 
```

But unfortunately, the config file is never read. I checked the source code in [`lws` package](https://github.com/lwsjs/lws/) and indeed, the config is only used at [`lib/command/serve.js`](https://github.com/lwsjs/lws/blob/24f979957776aac334da811b0a9f7dc03d51d30c/lib/command/serve.js#L240) which I believe is only invoked in the CLI application use case.

**Comment by 75lb** [comment]
interesting, thanks for reporting this.. yes, the `configFile` option should be honoured from both API and CLI use cases. Will look into this soon - thanks for the PR.

**Comment by 75lb** [comment]
Fixed and released in [v3.0.0](https://github.com/lwsjs/local-web-server/releases)
